packages/fontaine/src/index.ts

```
@@ -1,7 +1,72 @@
-export type { ResolveCategoryFallbacksOptions } from './fallbacks'
-export { getMetricsForFamily, readMetrics } from './metrics'
+/**
+ * @typedef {Object} FallbackOptions
+ * @property {string} fontName The main font name to fallback from.
+ * @property {number} [minFontSize=10] The minimum font size to ensure readability.
+ * @property {number} [maxFontSize=100] The maximum font size to limit fallback scaling.
+ */
 
-export { FontaineTransform } from './transform'
-export type { FontaineTransformOptions } from './transform'
+import { generateMetricsCSS } from './metrics';
```

ESLint

[error] Extra semicolon.

(style/semi)